srcpkgs/odin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'odin'
+pkgname=odin
+version=0.10.0
+revision=1
+archs="x86_64-musl x86_64"
+wrksrc="Odin-${version}"
+build_style="gnu-makefile"
+hostmakedepends="pkg-config clang llvm"
+depends="clang"
+short_desc="Fast. concise, readable, pragmatic and open programming language"
+maintainer="Logen Kain <logen@sudotask.com>"
+license="BSD"
+homepage="https://odin-lang.org"
+distfiles="https://github.com/odin-lang/Odin/archive/v${version}.tar.gz"
+checksum=04a1c9e8719281a598cac5e9cc3e5ca2316cc038a8b8f9ae43f514aba967c635
+nopie=1
+make_build_args+=" debug"
+
+do_build() {
```

Comment:
Let the `gnu-makefile` buildstyle handle `do_build()`: There is the `make_use_env` variable for doing exactly this `do_build()`.